papaparse.js

```diff
@@ -1516,7 +1517,11 @@ License: MIT
 				for (var i = 0; i < rows.length; i++)
 				{
 					row = rows[i];
-					cursor += row.length;
+					if (config.header && i === 0 && firstLine !== undefined) {
```

Comment:
Do we really need to test if confix header? firstline will be undefined if config.header is not set, so I think we can remove the `config.header` test

Reply:
Thanks. I fixed that.